src/cryptonote_basic/hardfork.cpp

```diff
@@ -75,10 +75,10 @@ static constexpr std::array devnet_hard_forks =
   hard_fork{ 12, 0,     3,  1599848400 },
   hard_fork{ 13, 0,     4,  1599848400 }, 
   hard_fork{ 15, 0,     5,  1599848400 },
-  hard_fork{ 16, 0,   100,  1599848400 },
-  hard_fork{ 17, 0,   151,  1599848400 },
-  hard_fork{ 18, 0,   152,  1599848400 },
-  hard_fork{ 19, 0,   153,  1599848400 },
+  hard_fork{ 16, 0,   199,  1599848400 },
+  hard_fork{ 17, 0,   251,  1599848400 },
+  hard_fork{ 18, 0,   252,  1599848400 },
+  hard_fork{ 19, 0,   253,  1599848400 },
```

Comment:
Is this change necessary?  This is going to hard break current devnet.

Reply:
not necessary but I made the local devnet scripts send a bunch of funds to new wallets for a node with 10 contributors so it needed extra funds. Will remove